docs/guides/primitive-input-output.ipynb

```diff
@@ -356,6 +356,19 @@
     "print(f'The expectation values measured from this PUB are: \\n{result[0].data.evs}')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### How the Estimator calculates the error bar\n",
```

Comment:
Tentative section header title - thoughts @kaelynj ? We also have the option to put the entire section in an admonition but would perhaps be overkill

Reply:
What about "How the Estimator calculates error"? A bit of a nitpick, but I think we should specify computing the error rather than "error bars"